assets/wizards/googleAdManager/index.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * Google Ad Manager Wizard.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, render, Fragment } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { withWizard, FormattedHeader, Handoff } from '../../components/src';
+
+/**
+ * External dependencies
+ */
+import ManageAdSlotsScreen from './views/manageAdSlotsScreen';
+import EditAdSlotScreen from './views/editAdSlotsScreen';
+import { HashRouter, Redirect, Route, Switch } from 'react-router-dom';
+
+/**
+ * AdSlots wizard for managing and setting up adSlots.
+ */
+class GoogleAdManagerWizard extends Component {
+
+	/**
+	 * Constructor.
+	 */
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			adSlots: [],
+		};
+	}
+
+	/**
+	 * wizardReady will be called when all plugin requirements are met.
+	 */
+	onWizardReady = () => {
+		this.refreshAdSlots();
+	};
+
+	/**
+	 * Get the latest adSlots info.
+	 */
+	refreshAdSlots( callback ) {
```

Comment:
`callback` doesn't seem to be used.

```suggestion
	refreshAdSlots() {
```